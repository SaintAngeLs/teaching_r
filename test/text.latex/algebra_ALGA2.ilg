This is makeindex, version 2.16 [TeX Live 2022] (kpathsea + Thai support).
Scanning input file algebra_ALGA2.idx...done (0 entries accepted, 0 rejected).
Nothing written in algebra_ALGA2.ind.
Transcript written in algebra_ALGA2.ilg.
