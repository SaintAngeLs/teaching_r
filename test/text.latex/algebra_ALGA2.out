\BOOKMARK [1][-]{section.1}{\376\377\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n\000\040\000t\000o\000\040\000P\000y\000t\000h\000o\000n}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000I\000n\000s\000t\000a\000l\000l\000a\000t\000i\000o\000n}{}% 2
\BOOKMARK [1][-]{section.3}{\376\377\000Y\000o\000u\000r\000\040\000F\000i\000r\000s\000t\000\040\000P\000y\000t\000h\000o\000n\000\040\000P\000r\000o\000g\000r\000a\000m\0001\0001\0001\0001}{}% 3
\BOOKMARK [1][-]{section.4}{\376\377\000Y\000o\000u\000r\000\040\000F\000i\000r\000s\000t\000\040\000P\000y\000t\000h\000o\000n\000\040\000P\000r\000o\000g\000r\000a\000m}{}% 4
\BOOKMARK [2][-]{subsection.4.1}{\376\377\000T\000h\000e\000\040\000p\000r\000i\000n\000t\000\050\000\051\000\040\000f\000u\000n\000c\000t\000i\000o\000n}{section.4}% 5
\BOOKMARK [2][-]{subsection.4.2}{\376\377\000U\000s\000i\000n\000g\000\040\000f\000-\000s\000t\000r\000i\000n\000g\000s\000\040\000w\000i\000t\000h\000\040\000t\000h\000e\000\040\000p\000r\000i\000n\000t\000\050\000\051\000\040\000f\000u\000n\000c\000t\000i\000o\000n}{section.4}% 6
\BOOKMARK [2][-]{subsection.4.3}{\376\377\000U\000s\000i\000n\000g\000\040\000f\000-\000s\000t\000r\000i\000n\000g\000s\000\040\000w\000i\000t\000h\000\040\000t\000h\000e\000\040\000p\000r\000i\000n\000t\000\050\000\051\000\040\000f\000u\000n\000c\000t\000i\000o\000n}{section.4}% 7
\BOOKMARK [2][-]{subsection.4.4}{\376\377\000U\000s\000i\000n\000g\000\040\000t\000h\000e\000\040\000p\000r\000i\000n\000t\000\050\000\051\000\040\000f\000u\000n\000c\000t\000i\000o\000n\000'\000s\000\040\000p\000r\000o\000p\000e\000r\000t\000i\000e\000s}{section.4}% 8
\BOOKMARK [2][-]{subsection.4.5}{\376\377\000U\000s\000i\000n\000g\000\040\000t\000h\000e\000\040\000p\000r\000i\000n\000t\000\050\000\051\000.\000f\000o\000r\000m\000a\000t\000\050\000\051\000\040\000f\000u\000n\000c\000t\000i\000o\000n\000\040\000i\000n\000\040\000P\000y\000t\000h\000o\000n}{section.4}% 9
\BOOKMARK [1][-]{section.5}{\376\377\000V\000a\000r\000i\000a\000b\000l\000e\000s\000\040\000a\000n\000d\000\040\000T\000y\000p\000e\000s}{}% 10
\BOOKMARK [2][-]{subsection.5.1}{\376\377\000N\000u\000m\000e\000r\000i\000c\000\040\000T\000y\000p\000e\000s}{section.5}% 11
\BOOKMARK [2][-]{subsection.5.2}{\376\377\000S\000e\000q\000u\000e\000n\000c\000e\000\040\000T\000y\000p\000e\000s}{section.5}% 12
\BOOKMARK [2][-]{subsection.5.3}{\376\377\000M\000a\000p\000p\000i\000n\000g\000\040\000T\000y\000p\000e}{section.5}% 13
\BOOKMARK [2][-]{subsection.5.4}{\376\377\000S\000e\000t\000\040\000T\000y\000p\000e\000s}{section.5}% 14
\BOOKMARK [2][-]{subsection.5.5}{\376\377\000B\000o\000o\000l\000e\000a\000n\000\040\000T\000y\000p\000e}{section.5}% 15
\BOOKMARK [2][-]{subsection.5.6}{\376\377\000B\000i\000n\000a\000r\000y\000\040\000T\000y\000p\000e\000s}{section.5}% 16
\BOOKMARK [1][-]{section.6}{\376\377\000O\000p\000e\000r\000a\000t\000o\000r\000s}{}% 17
\BOOKMARK [2][-]{subsection.6.1}{\376\377\000A\000r\000i\000t\000h\000m\000e\000t\000i\000c\000\040\000O\000p\000e\000r\000a\000t\000o\000r\000s}{section.6}% 18
\BOOKMARK [2][-]{subsection.6.2}{\376\377\000C\000o\000m\000p\000a\000r\000i\000s\000o\000n\000\040\000O\000p\000e\000r\000a\000t\000o\000r\000s}{section.6}% 19
\BOOKMARK [2][-]{subsection.6.3}{\376\377\000A\000s\000s\000i\000g\000n\000m\000e\000n\000t\000\040\000O\000p\000e\000r\000a\000t\000o\000r\000s}{section.6}% 20
\BOOKMARK [2][-]{subsection.6.4}{\376\377\000L\000o\000g\000i\000c\000a\000l\000\040\000O\000p\000e\000r\000a\000t\000o\000r\000s}{section.6}% 21
\BOOKMARK [1][-]{section.7}{\376\377\000C\000o\000n\000t\000r\000o\000l\000\040\000F\000l\000o\000w}{}% 22
\BOOKMARK [2][-]{subsection.7.1}{\376\377\000I\000f\000\040\000S\000t\000a\000t\000e\000m\000e\000n\000t\000s}{section.7}% 23
\BOOKMARK [3][-]{subsubsection.7.1.1}{\376\377\000T\000h\000e\000\040\000e\000l\000s\000e\000\040\000k\000e\000y\000w\000o\000r\000d}{subsection.7.1}% 24
\BOOKMARK [3][-]{subsubsection.7.1.2}{\376\377\000T\000h\000e\000\040\000e\000l\000i\000f\000\040\000k\000e\000y\000w\000o\000r\000d}{subsection.7.1}% 25
\BOOKMARK [2][-]{subsection.7.2}{\376\377\000W\000h\000i\000l\000e\000\040\000L\000o\000o\000p\000s}{section.7}% 26
\BOOKMARK [2][-]{subsection.7.3}{\376\377\000F\000o\000r\000\040\000L\000o\000o\000p\000s}{section.7}% 27
\BOOKMARK [3][-]{subsubsection.7.3.1}{\376\377\000I\000t\000e\000r\000a\000t\000i\000n\000g\000\040\000o\000v\000e\000r\000\040\000l\000i\000s\000t\000s}{subsection.7.3}% 28
\BOOKMARK [3][-]{subsubsection.7.3.2}{\376\377\000U\000s\000i\000n\000g\000\040\000t\000h\000e\000\040\000r\000a\000n\000g\000e\000\050\000\051\000\040\000f\000u\000n\000c\000t\000i\000o\000n}{subsection.7.3}% 29
\BOOKMARK [3][-]{subsubsection.7.3.3}{\376\377\000N\000e\000s\000t\000e\000d\000\040\000f\000o\000r\000\040\000l\000o\000o\000p\000s}{subsection.7.3}% 30
\BOOKMARK [3][-]{subsubsection.7.3.4}{\376\377\000T\000h\000e\000\040\000e\000n\000u\000m\000e\000r\000a\000t\000e\000\050\000\051\000\040\000f\000u\000n\000c\000t\000i\000o\000n}{subsection.7.3}% 31
\BOOKMARK [1][-]{section.8}{\376\377\000F\000u\000n\000c\000t\000i\000o\000n\000s}{}% 32
\BOOKMARK [2][-]{subsection.8.1}{\376\377\000C\000a\000l\000l\000i\000n\000g\000\040\000a\000\040\000F\000u\000n\000c\000t\000i\000o\000n}{section.8}% 33
\BOOKMARK [2][-]{subsection.8.2}{\376\377\000P\000a\000r\000a\000m\000e\000t\000e\000r\000s\000\040\000a\000n\000d\000\040\000A\000r\000g\000u\000m\000e\000n\000t\000s}{section.8}% 34
\BOOKMARK [2][-]{subsection.8.3}{\376\377\000R\000e\000t\000u\000r\000n\000\040\000V\000a\000l\000u\000e\000s}{section.8}% 35
\BOOKMARK [2][-]{subsection.8.4}{\376\377\000D\000e\000f\000a\000u\000l\000t\000\040\000P\000a\000r\000a\000m\000e\000t\000e\000r\000\040\000V\000a\000l\000u\000e}{section.8}% 36
\BOOKMARK [2][-]{subsection.8.5}{\376\377\000M\000u\000l\000t\000i\000p\000l\000e\000\040\000P\000a\000r\000a\000m\000e\000t\000e\000r\000s\000\040\000a\000n\000d\000\040\000A\000r\000g\000u\000m\000e\000n\000t\000s}{section.8}% 37
\BOOKMARK [2][-]{subsection.8.6}{\376\377\000V\000a\000r\000i\000a\000b\000l\000e\000-\000l\000e\000n\000g\000t\000h\000\040\000A\000r\000g\000u\000m\000e\000n\000t\000s}{section.8}% 38
\BOOKMARK [2][-]{subsection.8.7}{\376\377\000R\000e\000c\000u\000r\000s\000i\000v\000e\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000s}{section.8}% 39
\BOOKMARK [2][-]{subsection.8.8}{\376\377\000L\000a\000m\000b\000d\000a\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000s}{section.8}% 40
\BOOKMARK [1][-]{section.9}{\376\377\000O\000b\000j\000e\000c\000t\000-\000O\000r\000i\000e\000n\000t\000e\000d\000\040\000P\000r\000o\000g\000r\000a\000m\000m\000i\000n\000g}{}% 41
\BOOKMARK [2][-]{subsection.9.1}{\376\377\000C\000l\000a\000s\000s\000e\000s\000\040\000a\000n\000d\000\040\000O\000b\000j\000e\000c\000t\000s}{section.9}% 42
\BOOKMARK [2][-]{subsection.9.2}{\376\377\000I\000n\000h\000e\000r\000i\000t\000a\000n\000c\000e}{section.9}% 43
\BOOKMARK [2][-]{subsection.9.3}{\376\377\000E\000n\000c\000a\000p\000s\000u\000l\000a\000t\000i\000o\000n\000\040\000a\000n\000d\000\040\000D\000a\000t\000a\000\040\000H\000i\000d\000i\000n\000g}{section.9}% 44
\BOOKMARK [2][-]{subsection.9.4}{\376\377\000G\000e\000o\000m\000e\000t\000r\000i\000c\000\040\000F\000i\000g\000u\000r\000e\000s\000\040\000a\000n\000d\000\040\000M\000e\000t\000h\000o\000d\000s}{section.9}% 45
\BOOKMARK [1][-]{section.10}{\376\377\000T\000u\000r\000t\000l\000e\000\040\000l\000i\000b\000r\000a\000r\000y}{}% 46
\BOOKMARK [2][-]{subsection.10.1}{\376\377\000G\000e\000n\000e\000r\000a\000l\000\040\000o\000p\000e\000r\000a\000t\000i\000o\000n\000s\000\040\000o\000n\000\040\000t\000h\000e\000\040\000T\000u\000r\000t\000l\000e\000\040\000p\000s\000e\000u\000d\000o\000\040\000o\000b\000j\000e\000c\000t}{section.10}% 47
\BOOKMARK [2][-]{subsection.10.2}{\376\377\000C\000h\000a\000n\000g\000i\000n\000g\000\040\000t\000h\000e\000\040\000p\000e\000n\000\040\000c\000o\000l\000o\000r\000\040\000a\000n\000d\000\040\000f\000i\000l\000l\000i\000n\000g\000\040\000s\000h\000a\000p\000e\000s}{section.10}% 48
\BOOKMARK [2][-]{subsection.10.3}{\376\377\000C\000h\000a\000n\000g\000i\000n\000g\000\040\000t\000h\000e\000\040\000p\000e\000n\000\040\000s\000i\000z\000e}{section.10}% 49
\BOOKMARK [1][-]{section.11}{\376\377\000C\000o\000m\000p\000l\000e\000x\000\040\000P\000y\000t\000h\000o\000n\000\040\000P\000r\000o\000g\000r\000a\000m\000s}{}% 50
\BOOKMARK [2][-]{subsection.11.1}{\376\377\000F\000a\000c\000t\000o\000r\000i\000a\000l\000\040\000F\000u\000n\000c\000t\000i\000o\000n}{section.11}% 51
\BOOKMARK [2][-]{subsection.11.2}{\376\377\000F\000i\000b\000o\000n\000a\000c\000c\000i\000\040\000S\000e\000q\000u\000e\000n\000c\000e}{section.11}% 52
\BOOKMARK [2][-]{subsection.11.3}{\376\377\000F\000a\000c\000t\000o\000r\000i\000a\000l\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000\040\000w\000i\000t\000h\000\040\000D\000y\000n\000a\000m\000i\000c\000\040\000P\000r\000o\000g\000r\000a\000m\000m\000i\000n\000g}{section.11}% 53
\BOOKMARK [2][-]{subsection.11.4}{\376\377\000F\000i\000b\000o\000n\000a\000c\000c\000i\000\040\000S\000e\000q\000u\000e\000n\000c\000e\000\040\000w\000i\000t\000h\000\040\000D\000y\000n\000a\000m\000i\000c\000\040\000P\000r\000o\000g\000r\000a\000m\000m\000i\000n\000g}{section.11}% 54
\BOOKMARK [2][-]{subsection.11.5}{\376\377\000S\000o\000r\000t\000i\000n\000g\000\040\000A\000l\000g\000o\000r\000i\000t\000h\000m\000:\000\040\000B\000u\000b\000b\000l\000e\000\040\000S\000o\000r\000t}{section.11}% 55
\BOOKMARK [2][-]{subsection.11.6}{\376\377\000P\000r\000i\000m\000e\000\040\000N\000u\000m\000b\000e\000r\000s}{section.11}% 56
\BOOKMARK [1][-]{section.12}{\376\377\000D\000a\000t\000a\000\040\000S\000t\000r\000u\000c\000t\000u\000r\000e\000s\000\040\000i\000n\000\040\000P\000y\000t\000h\000o\000n}{}% 57
\BOOKMARK [2][-]{subsection.12.1}{\376\377\000L\000i\000s\000t\000s}{section.12}% 58
\BOOKMARK [2][-]{subsection.12.2}{\376\377\000T\000u\000p\000l\000e\000s}{section.12}% 59
\BOOKMARK [2][-]{subsection.12.3}{\376\377\000S\000e\000t\000s}{section.12}% 60
\BOOKMARK [2][-]{subsection.12.4}{\376\377\000D\000i\000c\000t\000i\000o\000n\000a\000r\000i\000e\000s}{section.12}% 61
\BOOKMARK [1][-]{section.13}{\376\377\000P\000r\000o\000j\000e\000c\000t\000s}{}% 62
\BOOKMARK [2][-]{subsection.13.1}{\376\377\000P\000r\000o\000j\000e\000c\000t\000\040\0001}{section.13}% 63
\BOOKMARK [3][-]{subsubsection.13.1.1}{\376\377\000P\000r\000o\000j\000e\000c\000t\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{subsection.13.1}% 64
\BOOKMARK [3][-]{subsubsection.13.1.2}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\050\000m\000a\000y\000\040\000b\000e\000\040\000m\000o\000d\000i\000f\000i\000e\000d\000\040\000b\000y\000\040\000u\000s\000e\000r\000\040\000a\000s\000\040\000t\000h\000e\000\040\000t\000a\000s\000k\000\040\000i\000s\000\040\000c\000r\000e\000a\000t\000i\000v\000e\000\040\000o\000n\000e\000\051}{subsection.13.1}% 65
\BOOKMARK [3][-]{subsubsection.13.1.3}{\376\377\000T\000h\000e\000\040\000o\000n\000e\000\040\000o\000f\000\040\000p\000o\000s\000s\000i\000b\000l\000e\000\040\000s\000o\000l\000u\000t\000i\000o\000n\000s}{subsection.13.1}% 66
\BOOKMARK [2][-]{subsection.13.2}{\376\377\000P\000r\000o\000j\000e\000c\000t\000\040\0002\000\040\000s\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n}{section.13}% 67
\BOOKMARK [3][-]{subsubsection.13.2.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.2}% 68
\BOOKMARK [3][-]{subsubsection.13.2.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.2}% 69
\BOOKMARK [3][-]{subsubsection.13.2.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.2}% 70
\BOOKMARK [2][-]{subsection.13.3}{\376\377\000P\000r\000o\000j\000e\000c\000t\000\040\0003\000\040\000s\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n}{section.13}% 71
\BOOKMARK [3][-]{subsubsection.13.3.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.3}% 72
\BOOKMARK [3][-]{subsubsection.13.3.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.3}% 73
\BOOKMARK [3][-]{subsubsection.13.3.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.3}% 74
\BOOKMARK [2][-]{subsection.13.4}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000t\000h\000e\000\040\000P\000r\000o\000j\000e\000c\000t\000\040\0004}{section.13}% 75
\BOOKMARK [3][-]{subsubsection.13.4.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.4}% 76
\BOOKMARK [3][-]{subsubsection.13.4.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.4}% 77
\BOOKMARK [3][-]{subsubsection.13.4.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.4}% 78
\BOOKMARK [2][-]{subsection.13.5}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000t\000h\000e\000\040\000P\000r\000o\000j\000e\000c\000t\000\040\0005}{section.13}% 79
\BOOKMARK [3][-]{subsubsection.13.5.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.5}% 80
\BOOKMARK [3][-]{subsubsection.13.5.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.5}% 81
\BOOKMARK [3][-]{subsubsection.13.5.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0002\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.5}% 82
\BOOKMARK [2][-]{subsection.13.6}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000t\000h\000e\000\040\000P\000r\000o\000j\000e\000c\000t\000\040\0006}{section.13}% 83
\BOOKMARK [3][-]{subsubsection.13.6.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.6}% 84
\BOOKMARK [3][-]{subsubsection.13.6.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0001\0005\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.6}% 85
\BOOKMARK [3][-]{subsubsection.13.6.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0001\0005\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.6}% 86
\BOOKMARK [3][-]{subsubsection.13.6.4}{\376\377\000T\000a\000s\000k\000\040\0004\000\040\000\050\0002\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.6}% 87
\BOOKMARK [3][-]{subsubsection.13.6.5}{\376\377\000T\000a\000s\000k\000\040\0005\000\040\000\050\0002\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.6}% 88
\BOOKMARK [2][-]{subsection.13.7}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000t\000h\000e\000\040\000P\000r\000o\000j\000e\000c\000t\000\040\0007}{section.13}% 89
\BOOKMARK [3][-]{subsubsection.13.7.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0002\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.7}% 90
\BOOKMARK [3][-]{subsubsection.13.7.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0002\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.7}% 91
\BOOKMARK [3][-]{subsubsection.13.7.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0002\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.7}% 92
\BOOKMARK [3][-]{subsubsection.13.7.4}{\376\377\000T\000a\000s\000k\000\040\0004\000\040\000\050\0002\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.7}% 93
\BOOKMARK [3][-]{subsubsection.13.7.5}{\376\377\000T\000a\000s\000k\000\040\0005\000\040\000\050\0002\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.7}% 94
\BOOKMARK [2][-]{subsection.13.8}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000t\000h\000e\000\040\000P\000r\000o\000j\000e\000c\000t\000\040\0008}{section.13}% 95
\BOOKMARK [3][-]{subsubsection.13.8.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0002\0005\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.8}% 96
\BOOKMARK [3][-]{subsubsection.13.8.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0002\0005\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.8}% 97
\BOOKMARK [3][-]{subsubsection.13.8.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0002\0005\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.8}% 98
\BOOKMARK [3][-]{subsubsection.13.8.4}{\376\377\000T\000a\000s\000k\000\040\0004\000\040\000\050\0002\0005\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.8}% 99
\BOOKMARK [2][-]{subsection.13.9}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000t\000h\000e\000\040\000P\000r\000o\000j\000e\000c\000t\000\040\0009}{section.13}% 100
\BOOKMARK [3][-]{subsubsection.13.9.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0002\0005\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.9}% 101
\BOOKMARK [3][-]{subsubsection.13.9.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0002\0005\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.9}% 102
\BOOKMARK [3][-]{subsubsection.13.9.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0002\0005\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.9}% 103
\BOOKMARK [3][-]{subsubsection.13.9.4}{\376\377\000T\000a\000s\000k\000\040\0004\000\040\000\050\0002\0005\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.9}% 104
\BOOKMARK [2][-]{subsection.13.10}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000t\000h\000e\000\040\000P\000r\000o\000j\000e\000c\000t\000\040\0001\0000}{section.13}% 105
\BOOKMARK [3][-]{subsubsection.13.10.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0002\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.10}% 106
\BOOKMARK [3][-]{subsubsection.13.10.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0002\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.10}% 107
\BOOKMARK [3][-]{subsubsection.13.10.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0003\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.10}% 108
\BOOKMARK [3][-]{subsubsection.13.10.4}{\376\377\000T\000a\000s\000k\000\040\0004\000\040\000\050\0003\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.13.10}% 109
\BOOKMARK [1][-]{section.14}{\376\377\000A\000d\000d\000i\000t\000i\000o\000n\000a\000l\000\040\000r\000e\000f\000e\000r\000e\000n\000c\000e\000s}{}% 110
\BOOKMARK [2][-]{subsection.14.1}{\376\377\000A\000\040\000D\000e\000e\000p\000\040\000D\000i\000v\000e\000\040\000i\000n\000t\000o\000\040\000t\000h\000e\000\040\000Q\000u\000a\000d\000r\000a\000t\000i\000c\000\040\000E\000q\000u\000a\000t\000i\000o\000n}{section.14}% 111
\BOOKMARK [2][-]{subsection.14.2}{\376\377\000S\000o\000l\000v\000i\000n\000g\000\040\000S\000e\000c\000o\000n\000d\000\040\000O\000r\000d\000e\000r\000\040\000E\000q\000u\000a\000t\000i\000o\000n\000s}{section.14}% 112
